src/main/kotlin/no/nav/syfo/consumer/distribuerjournalpost/JournalpostdistribusjonConsumer.kt

```diff
@@ -50,4 +55,43 @@ class JournalpostdistribusjonConsumer(urlEnv: UrlEnv, private val azureAdTokenCo
             )
         }
     }
+
+    suspend fun distribuerTvungetJournalpost(
```

Comment:
Blir vel kun tvingSentralPrint variabelen som er ulik - hadde det gått an å gjenbruke noe kode her, for eksempel ved å trekke felles ting til felles funksjon, evnt bare ta tvingSentralPrint som en parameter i opprinnelig funksjon?